gluten-core/src/main/scala/org/apache/gluten/GlutenPlugin.scala

```diff
@@ -118,6 +118,7 @@ private object GlutenDriverPlugin extends Logging {
             // 1GB default
             1024 * 1024 * 1024
           }
+        conf.set(GlutenCoreConfig.SPARK_OFFHEAP_SIZE_KEY, "0")
```

Comment:
@zhli1142015 this config is required during gluten loading native libs

Reply:
> When DYNAMIC_OFFHEAP_SIZING_ENABLED is enabled, the memory used by Velox is tracked under the on-heap category by the Spark memory manager, so Ithink there is no need to enable off-heap memory.

Thanks, that makes sense to me. @zhouyuan Can we point this out in the documentation?